react-function-components/src/index.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+function CustomButton(props) {
+  return <button>Click {props.type}</button>;
+}
+
+const element = <CustomButton type="Me!" />;
+ReactDOM.render(
+  element,
+  document.querySelector('#root')
+);
```

Comment:
You don't have to save the JSX into a variable, you can just directly use the component in the `render` method. For example:

```JSX
ReactDOM.render(
  <CustomButton />,
  document.querySelector('#root')
);
```